#CREATE request parameters
string[] action
float32[] target
float32 iterations
--------------------------------
#CREATE response parameters
bool walking
